kolibri/core/theme_hook.py

```diff
@@ -0,0 +1,165 @@
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import logging
+
+from kolibri.plugins import hooks
+import kolibri
+from django.utils.six.moves.urllib import parse
+from django.conf import settings
+import os
+
+logger = logging.getLogger(__name__)
+
+
+# Important for cache busting
+THEME_NAME = "themeName"
+THEME_VERSION = "themeVersion"
+
+# These constants are used by theme.js and the $theme mixin on the front-end
+TOKEN_MAPPING = "tokenMapping"
+BRAND_COLORS = "brandColors"
+PRIMARY = "primary"
+SECONDARY = "secondary"
+COLOR_V50 = "v_50"
+COLOR_V100 = "v_100"
+COLOR_V200 = "v_200"
+COLOR_V300 = "v_300"
+COLOR_V400 = "v_400"
+COLOR_V500 = "v_500"
+COLOR_V600 = "v_600"
+COLOR_V700 = "v_700"
+COLOR_V800 = "v_800"
+COLOR_V900 = "v_900"
+SIGN_IN = "signIn"
+SIDE_NAV = "sideNav"
+APP_BAR = "appBar"
+BACKGROUND = "background"
+TITLE = "title"
+TOP_LOGO = "topLogo"
+IMG_SRC = "src"
+IMG_STYLE = "style"
+IMG_ALT = "alt"
+SHOW_K_FOOTER_LOGO = "showKolibriFooterLogo"
+
+# This is the image file name that will be used when customizing the sign-in background
+# image using the 'kolibri manage background' command. It does not attempt to use a file
+# extension (like .jpg) because we don't know if it's a JPG, SVG, PNG, etc...
+DEFAULT_BG_IMAGE_FILE = "background_image"
```

Comment:
I assume most browsers are smart enough to not need the file extension for the mimetype hinting.

Reply:
yes, I believe so